src/shared/inspect.ts

```diff
@@ -31,11 +33,12 @@ export const defaults = {
   COLOR: true
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
question: is this likely to be needed often? can the ignore be added to the config file? or is this a case where a global ignore might prove detrimental?